test/test-get.js

```diff
@@ -5,19 +5,22 @@ var nodeunit = require('nodeunit');
 var imdb = require('../lib/imdb.js');
 
 module.exports.testGetSuccessful = function(test) {
```

Comment:
While this is a great start, this should be it's own test, leaving the original untouched if possible